bib/cmd/bootc-image-builder/image.go

```diff
@@ -154,6 +156,195 @@ func pipelinesForDiskImage(c *ManifestConfig, rng *rand.Rand) (image.ImageKind,
 	return img, nil
 }
 
+func pipelinesForISO(c *ManifestConfig, rng *rand.Rand) (image.ImageKind, error) {
+	if c.Imgref == "" {
+		fail("pipeline: no base image defined")
+	}
+	ref := "ostree/1/1/0"
+	tlsVerify := true
+	containerSource := container.SourceSpec{
+		Source:    c.Imgref,
+		Name:      c.Imgref,
+		TLSVerify: &tlsVerify,
+	}
+
+	img := image.NewAnacondaContainerInstaller(containerSource, ref)
```

Comment:
I hope to do a proper review later, but something I need to get off my head right now:

Can we switch to lz4 for now?
```
img.SquashfsCompression = "lz4"
```

I think that we should focus on speed first, and deal with artifact size later.

Reply:
Agreed.

Reply:
Ok, I had this in the PR but had to drop it again as our dracut that mounts the squashfs is without lz4 support. I'll take a second look :)

Reply:
Right, I've now set it to `zstd` which is both faster and supported by dracut for the squashfs.